local-install-siva-java.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Why do you need to do this for siva-java and enry-java? I think it should be done using https://jitpack.io